access-control-app/build-access-control-for-container.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+lein uberjar
+docker build -t access-control-app .
+docker run access-control-app
```

Comment:
Don't we need a `-p 3011:3011` to make port 3011 from the container available at port 3011 on localhost?